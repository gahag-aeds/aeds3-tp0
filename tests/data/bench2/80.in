80 40
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 
Add 43 68
Max 23 48
Add 5 30
Sum 45 70
Sub 18 43
Sum 39 64
Sub 40 65
Sum 21 46
Add 45 70
Min 52 77
Add 25 50
Max 14 39
Sub 18 43
Max 17 42
Add 7 32
Sum 51 76
Add 15 40
Max 11 36
Sub 10 35
Max 41 66
Sub 14 39
Sum 6 31
Sub 30 55
Min 16 41
Sub 30 55
Min 52 77
Sub 8 33
Max 38 63
Add 33 58
Max 29 54
Add 15 40
Min 22 47
Add 51 76
Max 20 45
Sub 12 37
Max 14 39
Sub 20 45
Min 13 38
Sub 34 59
Sum 27 52
