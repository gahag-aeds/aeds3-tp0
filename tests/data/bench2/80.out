94
981
948
1184
2
95
94
1075
96
99
1347
4
1
99
95
2
94
93
3
1111
